src/main/java/com/org/jenkins/custom/jenkins/distribution/service/PackagerController.java

```diff
@@ -34,4 +40,23 @@
             return new ResponseEntity(HttpStatus.NOT_FOUND);
         }
     }
+
+    @PostMapping (path = "/downloadWarPackage")
+    public ResponseEntity<?> downloadPackageConfig(@RequestBody String postPayload) {
+        LOGGER.info("Request Received for downloading war file with configuration" + postPayload);
+        try {
+            return new PackagerDownloadService().downloadWAR(getWarVersion(), postPayload);
+        } catch (Exception e) {
+            LOGGER.severe(e.toString());
+            e.printStackTrace();
```

Comment:
Please avoid print the exception stack into stdio directly. You can choose a better logger framework which can allow to print the exception message with stack trace.

Reply:
Yep got it :+1: